openwisp_controller/config/tests/test_selenium.py

```diff
@@ -130,6 +132,48 @@ def test_create_new_device(self):
             'The Device “11:22:33:44:55:66” was added successfully.',
         )
 
+    def test_device_preview_keyboard_shortcuts(self):
+        self._create_config(device=self._create_device(name='Test'))
+        self.login()
+        self.open(reverse('admin:config_device_changelist'))
+
+        # open first device in the list
+        self.web_driver.find_element(
+            by=By.CSS_SELECTOR, value='tbody tr:nth-child(1) th a'
+        ).click()
```

Comment:
This can be shortened by directly opening the device page.
   
```suggestion
        device = self._create_config(device=self._create_device(name='Test')).device
        self.login()
        self.open(reverse('admin:config_device_change', args=[device.id]))
```